.github/workflows/analytics-metadata-updater.yml

```diff
@@ -0,0 +1,56 @@
+name: CDK Analytics Metadata Updater
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 16 * * 2" # Every week at 10 AM PDT or 9 AM PST
```

Comment:
why not with each PR ?

Reply:
this also works, just a question .. also we need to make sure that we call before any release

Reply:
Yes, I think as long as we run this before release, it should be fine. There is no need to run every PR since this workflow is not a small job and take time and resource to run.